src/widgets/dialogs/BadgePickerDialog.cpp

```diff
@@ -0,0 +1,71 @@
+#include "BadgePickerDialog.hpp"
+#include <QSizePolicy>
+#include "singletons/Resources.hpp"
+
+#include "providers/twitch/TwitchBadges.hpp"
```

Comment:
What's with the weird order of those includes?

Reply:
We don't have a standard of include order, so I'll mark this resolved for now.